.rubocop.yml

```diff
@@ -8,6 +8,11 @@ AllCops:
     - .vendor/**/*
     - pkg/**/*
     - spec/fixtures/**/*
+
+RSpec/MultipleExpectations:
```

Comment:
this will be overwritten by our modulesync config(https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.rubocop.yml). Why did you disable it? In general we try to fix MultipleExpectations errors.